Sources/TransferMethod/SelectTransferMethodTypeCell.swift

```diff
@@ -16,11 +16,12 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+import HyperwalletSDK
 import UIKit
 
 /// Represents the country and currency data to be displyed on the CountryCurrencyCell
 struct SelectTransferMethodTypeConfiguration {
-    let transferMethodType: String
+    let transferMethodType: HyperwalletTransferMethodType
```

Comment:
This should still be String. let transferMethodTypeName: String

Reply:
fixed